docs/querying/using-caching.md

```diff
@@ -0,0 +1,91 @@
+---
+id: using-caching
+title: "Using query caching"
+---
+
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one
+  ~ or more contributor license agreements.  See the NOTICE file
+  ~ distributed with this work for additional information
+  ~ regarding copyright ownership.  The ASF licenses this file
+  ~ to you under the Apache License, Version 2.0 (the
+  ~ "License"); you may not use this file except in compliance
+  ~ with the License.  You may obtain a copy of the License at
+  ~
+  ~   http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+
+This topic covers how to configure services to populate and use the Druid query cache. For a conceptual overview and use cases, see [Query caching](./caching.md). For information on how to configure the caching mechanism, see [Cache configuration](../configuration/index.md#cache-configuration).
+
+All query caches have a pair of parameters that control the way individual queries interact with the cache:
+
+- `useCache` to instruct queries to use the cache for results.
+- `populateCache` to instruct a query to cache its results.
+
+The separation of concerns, usage and population, lets you include cached results for queries on uncommon data without polluting the cache with results that are unlikely to be reused by other queries, for example, large reports or queries on very old data.
+
+To use caching, it must be enabled in the settings for at least one service in the runtime properties. By default, per-query cache is enabled on Historicals. For individual queries, you can control cache usage and population within the query context.
```

Comment:
nit: When I read this, I think that if it's enabled on 1 service, it applies to all of my cluster.

```suggestion
To use caching, it must be enabled in the settings for the service where caching should be enabled in the runtime properties. By default, per-segment cache is enabled on Historicals. For individual queries, you can control cache usage and population within the query context.
```

^ it should say `per-segment`, not `per-query` (that's a little too close to whole query caching 😅 )

Reply:
```suggestion
To use caching, it must be enabled in the settings for the service to perform caching in the service's runtime properties. By default, per-segment cache is enabled on Historicals. For individual queries, you can control cache usage and population within the query context.
```